### PR TITLE
Add examples for the keywords array in Pulumi schema

### PR DESCRIPTION
**Pull request by praneetloke**

### CODE REVIEW

File: themes/default/content/docs/guides/pulumi-packages/schema.md [hunk 1]
```diff
@@ -106,7 +106,10 @@ Complete schema examples that include a much wider range of schema configuration
...
 | `version`           | `string`                                   | Yes      | Version is the version of the package. The version must be valid semver.                                                          |
 | `description`       | `string`                                   | No       | Description is the description of the package.                                                                                    |
-| `keywords`          | `array[string]`                            | No       | Keywords is the list of keywords that are associated with the package, if any.                                                    |
+| `keywords`          | `array[string]`                            | No       | Keywords is the list of keywords that are associated with the package, if any. 
+Some reserved keywords can be specified as well that help with categorizing the package in the Pulumi registry. `category/<name>` and `kind/<type>` are the only reserved keywords at this time,
+where `<name>` can be one of: `cloud`, `database`, `infrastructure`, `network`, `utility`, `vcs` and `<type>` is either `native` or `component`. If the package is a TF-based provider, then don't include a
```
Comment (infin8x) [inline themes/default/content/docs/guides/pulumi-packages/schema.md:111]:
```suggestion
where `<name>` can be one of: `cloud`, `database`, `infrastructure`, `network`, `utility`, `vcs` and `<type>` is either `native` or `component`. If the package is a bridged Terraform provider, then don't include a
```

**Review by infin8x — APPROVED** [review]
🚀 it! One small suggestion.

**Comment by praneetloke** [comment]
Hmm I am trying to get the updated sentence show up in the same table cell. Right now, my line breaks are treated as new table rows. I have pushed an update to use `<br>` that will hopefully render the contents in one cell.

**Comment by praneetloke** [comment]
Dusting off this PR.

**Pull request merged**